.appveyor.yml

```diff
@@ -23,6 +23,7 @@ clone_depth: 50
 install:
   - git submodule update --init --recursive
   - npm install -g npm@6.4.0 yarn@1.9.4
+  - npm install -g bower@1.8.2
```

Comment:
Simply add this to the previous line (as a first item, sorted alphabetically), so `npm install` is only called once.

Reply:
Done.